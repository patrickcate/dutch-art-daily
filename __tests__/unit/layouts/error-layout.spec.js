import ErrorLayout from '@layouts/error-layout.vue'

describe('ErrorLayout', () => {
  let wrapper
  let options

  beforeEach(() => {
    options = {
      mocks: {
        $route: {
          params: {
            date: '13-13',
          },
        },
      },
    }

    wrapper = createWrapper(ErrorLayout, options)
  })

  it('should be a Vue instance', () => {
    expect(wrapper.isVueInstance()).toBe(true)
  })

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
