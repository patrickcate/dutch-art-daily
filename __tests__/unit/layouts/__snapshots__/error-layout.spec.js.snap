// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorLayout renders correctly 1`] = `
<div class="l-page l-page--error">
  <div class="background-stack" style="background-color: rgb(66, 50, 33); background-image: url(/photos/01-02/01-02--lqi-8BAoia4yi0O.jpg);">
    <div class="background-stack__item background-stack__item--today" style="background-color: rgb(66, 50, 33); background-image: url(/photos/01-02/01-02--lqi-8BAoia4yi0O.jpg);"></div>
  </div>
  <div class="l-page__content">
    <div class="l-page__header">
      <div><a href="/" rel="home" class="home"><svg width="131px" height="50px" class="base-icon" aria-label="Home">
            <use></use>
          </svg></a></div>
    </div>
    <nuxt-stub></nuxt-stub>
    <the-footer-stub class="l-page__footer"></the-footer-stub>
  </div>
</div>
`;
